vpr/src/place/place.cpp

```diff
@@ -2313,6 +2314,15 @@ static void get_non_updateable_bb(ClusterNetId net_id, t_bb* bb_coord_new) {
 
     ClusterBlockId bnum = cluster_ctx.clb_nlist.net_driver_block(net_id);
     pnum = net_pin_to_tile_pin_index(net_id, 0);
+
+    if (unsigned(pnum) > (physical_tile_type(bnum)->pin_width_offset.size() - 1)) {
```

Comment:
Comment issue open for a better fix -- give issue number. Explain why this code is needed.

Reply:
added better fix for bug - sync pins if needed in apply_move_blocks and revert_move_blocks